libnatspec/natspecjs/package.json

```diff
@@ -1,24 +0,0 @@
-{
```

Comment:
I've created a new issue for that future TODO:
https://github.com/ethereum/webthree-umbrella/issues/648